index.html

```diff
@@ -1021,8 +1021,17 @@ <h2>Thank you for your Feedback! 💚</h2>
 </script>
     
   </main><!-- End #main -->
+  <br>
+  <link rel="stylesheet" href="./assets/css/visitors.css">
   <!-- ############### Footer ############### -->
-  
+  <div class="visitor-counter">
+    <div>Visitor</div>
+    <div class="website-counter"></div>
+  </div>
+  <script src="path/to/visitorCounter.js"></script>

```

Comment:
I think this path need to be updated

Reply:
Why ? @varshith

Reply:
Where is path/to/ .... Directory in our project?